### PR TITLE
Fix logo covering alert message issue #167303268

### PR DESCRIPTION
**Pull request by akegan**
https://www.pivotaltracker.com/story/show/167303268

No pattern lib updates in this PR because updates were pulled in #1291

### CODE REVIEW

**Comment by akegan** [comment]
@cpalmatier I tagged you for review on this in case you feed comfortable reviewing these changes. if not I can see if Logan can review it.

File: app/assets/javascripts/shared/SharedController.js.coffee [hunk 1]
```diff
@@ -27,7 +27,7 @@ SharedController = ($scope, $state, $stateParams, $window, SharedService, Extern
 
   $scope.listingEmailAlertUrl = "http://eepurl.com/dkBd2n"
 
-  $scope.alertMessage = if $window.ALERT_MESSAGE then $window.ALERT_MESSAGE else ''
```
Comment (akegan) [inline app/assets/javascripts/shared/SharedController.js.coffee]: this fixes the issue where apostrophes were showing up as &#39;

**Comment by cpalmatier** [comment]
@akegan this looks good to me although i am def making some assumptions re: 
76771d5 as I am not as well-versed in coffeescript as i'd like to be. up to you whether you want to bring in Logan.

**Comment by akegan** [comment]
ok, if you're up for checking out the review instructions and doing them that would be great. I'll get some input from Sasha and others on the eng team to make sure i'm not exposing us to any security vulns withe the unescape update

**Comment by cpalmatier** [comment]
@akegan i will run through the review instructions on the review app this evening.

**Comment by akegan** [comment]
sasha checked out the unescaped bit and it seems like we'll be OK with that code change, so once you check out the review instructions and approve you should be good to merge.

**Pull request merged**